MH4U Dex/RegionEncylcopediaCollectionViewCell.m

```diff
@@ -8,6 +8,26 @@
 
 #import "RegionEncylcopediaCollectionViewCell.h"
 
+@interface RegionEncylcopediaCollectionViewCell ()
+
+@property (nonatomic, weak) IBOutlet UILabel *regionNameLabel;
+@property (nonatomic, weak) IBOutlet UIImageView *regionIconImageView;
+
+@end
+
 @implementation RegionEncylcopediaCollectionViewCell
 
+- (void)displayContents
+{
+    NSString *imageName = [NSString stringWithFormat:@"%@-icon", self.regionKeyName];
+    UIImage *image = [UIImage imageNamed:imageName];
+    if (image) {
+        self.regionIconImageView.image = image;
+    } else {
+        self.regionIconImageView.image = nil;
+    }
```

Comment:
That's true, but for me at least, writing it this way makes it seem clear to me that leaving the image blank is unambiguously what should happen if the image is not found.